src/turnkeyml/analyze/script.py

```diff
@@ -749,7 +761,17 @@ def forward_spy(*args, **kwargs):
             if invocation_hash not in model_info.unique_invocations:
                 model_info.unique_invocations[invocation_hash] = (
                     util.UniqueInvocationInfo(
-                        hash=invocation_hash,
+                        name=model_info.name,
+                        script_name=model_info.script_name,
+                        file=model_info.file,
+                        line=model_info.line,
+                        params=model_info.params,
+                        depth=model_info.depth,
+                        build_model=model_info.build_model,
+                        model_type=model_info.model_type,
+                        model_class=type(model_info.model),
+                        invocation_hash=invocation_hash,
+                        hash=model_info.hash,
```

Comment:
If anyone knows a better way to bulk-copy this info from `model_info` let me know...